pkg/controller/dgraph/models/pod.go

```diff
@@ -73,10 +74,21 @@ func StorePod(k8sPod api_v1.Pod) error {
 		uid = assigned.Uids["blank-0"]
 	}
 
-	pod = Pod{
-		ID:         dgraph.ID{Xid: xid, UID: uid},
-		Containers: StoreAndRetrieveContainers(k8sPod, uid),
+	podDeletedTimestamp := k8sPod.GetDeletionTimestamp()
+	isDeleted := !podDeletedTimestamp.IsZero()
+	if isDeleted {
```

Comment:
use isDeleted flag just to update endtime.

Reply:
Done